Code/GraphMol/MolOps.cpp

```diff
@@ -183,62 +184,104 @@ void halogenCleanup(RWMol &mol, Atom *atom) {
   }
 }
 
+bool isMetal(const Atom *atom) {
+  // This is the list of not metal atoms from QueryOps.cpp
+  static const std::set<int> notMetals{0,  1,  2,  5,  6,  7,  8,  9,
+                                       10, 14, 15, 16, 17, 18, 33, 34,
+                                       35, 36, 52, 53, 54, 85, 86};
```

Comment:
rather than duplicating this here, how about just directly using a query atom built with `makeMAtomQuery()`?

If there is a reason to duplicate it (perhaps performance), then I'd prefer to have a `vector` of nonMetals defined in `QueryOps.h` and used here and in `QueryOps.cpp`

Reply:
The reason for having a separate list was the suspicion that at some point we would want to refine the list of metals that this cleanup would apply to.  But I guess that bridge should be crossed later.  Premature optimisation and all that...